server/tests/parser_tests.py

```diff
@@ -45,33 +46,49 @@ def _test_case_parser(test_case):
 def test_parse():
```

Comment:
break this into two test methods `def test_parse_ios():` & `def test_parse_android():`